Use Control Plane instead of Master (#18438)
kubernetes/website: content/en/docs/reference/glossary/kube-scheduler.md

date: 2018-04-12
full_link: /docs/reference/generated/kube-scheduler/
short_description: >

>   Component of the Control Plane that watches newly created pods that have no node assigned, and selects a node for them to run on.

Reviewer: ```suggestion
  Control Plane component that watches for newly created pods with no assigned node, and selects a node for them to run on.
```

aka: 
tags:
- architecture
---

>  Component of the Control Plane that watches newly created pods that have no node assigned, and selects a node for them to run on.

Reviewer: ```suggestion
 Control Plane component that watches for newly created pods with no assigned node, and selects a node for them to run on.
```
